src/components/storage/CockpitStorageIntegration.scss

```diff
@@ -57,3 +60,25 @@ ul.cockpit-storage-integration-requirements-hint-list {
         color: unset;
     }
 }
+
+// Simulate Backdrop behavior for modals in the iframe
+%iframe-zindex {
+  position: relative;
+  z-index: 10;
+}
+
+iframe {
+  @extend %iframe-zindex;
+}
+
+// Simulate a PF background scrim
+.cockpit-has-modal {
+  @extend %iframe-zindex;
+
+  &::after {
+    @extend .pf-v5-c-backdrop;
+    --pf-v5-c-backdrop--Position: absolute;
+    content: "";
+    z-index: 9;
```

Comment:
This block changes PF's position variable to always be absolute (which is what we want — it needs to have position to have a z-index take effect and it should be positioned absolutely (relative to the parent with position, as usual)). It then changes the content to blank, and adjusts the z-index to 9 to be below the iframe (which is at 10, as specified above).